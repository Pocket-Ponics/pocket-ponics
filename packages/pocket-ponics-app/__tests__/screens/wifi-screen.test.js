import React from 'react'
import { shallow } from 'enzyme'

import WifiScreen from '../../screens/wifi-screen'

describe('Wifi Screen', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<WifiScreen navigation={{ getParam: jest.fn() }}/>
		).html()
		expect(tree).toMatchSnapshot()
	})
})