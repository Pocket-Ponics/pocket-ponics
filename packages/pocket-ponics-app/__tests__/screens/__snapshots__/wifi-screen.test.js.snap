// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wifi Screen renders correctly 1`] = `"<View style=\\"flex:1\\"><View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><Text style=\\"color:#FFFFFF;font-size:32px;margin-bottom:20px;text-align:center\\">Select a WiFi network</Text><RCTScrollView style=\\"margin-bottom:20px\\" initialNumToRender=\\"10\\" maxToRenderPerBatch=\\"10\\" scrollEventThrottle=\\"50\\" updateCellsBatchingPeriod=\\"50\\" windowSize=\\"21\\" numColumns=\\"1\\" viewabilityConfigCallbackPairs=\\"\\" stickyHeaderIndices=\\"\\"><View></View></RCTScrollView><View style=\\"align-items:center;opacity:1\\"><Text style=\\"color:#84AB71;font-size:22px\\">Cancel Setup</Text></View></View></View>"`;
